doc-content/enterprise-only/cases/case-management-case-key-prefix-proc.adoc

```diff
@@ -0,0 +1,94 @@
+[id='case-management-case-key-prefix-proc-{context}']
+= Configuring case ID prefixes
+
+The `caseId` parameter is a string value that is the identifier of the case instance. You can configure the `Case ID prefix` to distinguish different types of cases. The prefix is followed by a generated ID in the format `ID-XXXXXXXXXX`, where `XXXXXXXXXX` is a generated number that provides a unique ID for the case instance. If a prefix is not provided, the default prefix is `CASE` with the following identifiers:
+
+`CASE-0000000001`
+
+`CASE-0000000002`
+
+`CASE-0000000003`
+
+You can specify any prefix. For example, if you specify the prefix `IT`, the following identifiers are generated:
+
+`IT-0000000001`
+
+`IT-0000000002`
+
+`IT-0000000003`
+
+In addition to specifying a prefix, you can customize the expression for generating the `caseId` using MVEL templates, which include the following functions:
+
+* *LPAD* - Left padding
+* *RPAD* - Right padding
+* *TRUNCATE* - Truncate to the left
+* *UPPER* - Upper case
+
+The following examples show how to implement the available functions:
+
+[source]
+----
+<tns:metaValue>@{TRUNCATE(type,9)}</tns:metaValue>
+
+<tns:metaValue>@{LPAD(type,8,0)}</tns:metaValue>
+
+<tns:metaValue>@{RPAD(type,6,' ')}</tns:metaValue>
+----
```

Comment:
Missing an example for UPPER for completeness: 
`<tns:metaValue>@{UPPER(type)}</tns:metaValue>`

Perhaps it can be also mentioned that _type_ is a case file variable, which is passed when starting the case.

Reply:
Added UPPER example